src/search.cpp

```diff
@@ -345,12 +340,7 @@ Xapian::Enquire& Search::getEnquire() const
         return *mp_enquire;
     }
 
-    Xapian::QueryParser* queryParser = new Xapian::QueryParser();
-    if (m_query.m_verbose) {
-      std::cout << "Setup queryparser using language " << mp_internalDb->m_language << std::endl;
-    }
-    mp_internalDb->setupQueryparser(queryParser, m_query.m_suggestionMode);
-
+    auto queryParser = mp_internalDb->getQueryParser();
```

Comment:
This noted issue was already addressed in a later commit. 

~The type of this variable changed from a pointer-to-`Xapian::QueryParser` to simply `Xapian::QueryParser` (but did you mean to include a reference after `auto`? or maybe better make `getQueryParser()` return by value in order to avoid using the same `QueryParser` object for simultaneously parsing different queries in a multi-threaded environment?). Yet nothing was updated related to the usage of this variable. In particular 20 lines below one reads:~

```c++
delete queryParser;
```

~Obviously `Xapian::QueryParser` is convertible to a pointer to itself.~